config/settings.go

```diff
@@ -22,13 +22,14 @@ import (
 
 // AzureSettings represents all configurable data that is used to setup the Cloud Agent.
 type AzureSettings struct {
-	TenantID                     string `json:"tenantId"`
-	ConnectionString             string `json:"connectionString"`
-	SASTokenValidity             string `json:"sasTokenValidity"`
-	MessageMapperConfig          string `json:"messageMapperConfig"`
-	AllowedLocalTopicsList       string `json:"allowedLocalTopicsList"`
-	AllowedCloudMessageTypesList string `json:"allowedCloudMessageTypesList"`
-	IDScope                      string `json:"idScope"`
+	TenantID         string `json:"tenantId"`
+	ConnectionString string `json:"connectionString"`
+	SASTokenValidity string `json:"sasTokenValidity"`
+	IDScope          string `json:"idScope"`
+
+	// TODO: issue #19 - move passthrough topic settings outside AzureSettings
+	PassthroughCommandTopic   string `json:"passthroughCommandTopic"`
```

Comment:
- the name should be: ...Topics
- the type should be string[] - we already have such examples in suite bootstrapping and software update components